app/utils/tests/injectReducer.test.js

```diff
@@ -66,3 +67,32 @@ describe('injectReducer decorator', () => {
     expect(children.props).toEqual(props);
   });
 });
+
+describe('useInjectReducer hook', () => {
```

Comment:
i LOVE you for this test

Reply:
Made possible by @Mensae's work on migrating to react-testing-library

Reply:
wtb @Mensae !!!